### PR TITLE
Support CT-Aquifer Dimensionless P/T Summary Vectors

### PR DESCRIPTION
**Pull request by bska**
This commit adds support for calculating and outputting the CT-specific summary vectors "AAQTD" (dimensionless time, real time divided by aquifer's time constant) and "AAQPD" (dimensionless pressure-influence function evaluated at dimensionless time).

### CODE REVIEW

**Comment by bska** [comment]
Note that while useful in their own right, these quantities are also needed for restart output (for Carter-Tracy aquifers).

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this opm-simulators=3125 please

**Review by joakim-hove — APPROVED** [review]

**Comment by bska** [comment]
jenkins build this opm-simulators=3125 please

**Comment by bska** [comment]
jenkins build this opm-simulators=3125 please

File: opm/output/data/Aquifer.hpp [hunk 1]
```diff
@@ -34,81 +32,128 @@ namespace Opm { namespace data {
...
+        int aquiferID = 0;         //< One-based ID, range 1..NANAQ
+        double pressure = 0.0;     //< Aquifer pressure
+        double fluxRate = 0.0;     //< Aquifer influx rate (liquid aquifer)
+        double volume = 0.0;       //< Produced liquid volume
+        double initPressure = 0.0; //< Aquifer's initial pressure
+        double datumDepth = 0.0;   //< Aquifer's pressure reference depth
 
         AquiferType type;
         std::shared_ptr<FetkovichData> aquFet;
+        std::shared_ptr<CarterTracyData> aquCT;
```
Comment (joakim-hove) [inline opm/output/data/Aquifer.hpp:62]: I have a feeling `std::optional<>` is more "correct" than `std::shared_ptr<>`, or maybe `std::variant<>`?

Comment (bska) [reply opm/output/data/Aquifer.hpp:62]: > have a feeling `std::optional<>` is more "correct" than `std::shared_ptr<>`, or maybe `std::variant<>`?

Yes, probably, but that's a bigger refactoring which affects the restart file reader too.  I'd rather do that as a separate PR later.

**Comment by bska** [comment]
jenkins build this opm-simulators=3125 please

**Pull request merged**